### PR TITLE
Add icon to dropdown menu on iOs

### LINKED ISSUE #131: iOS Select dropdown missing icon
image

Select dropdown on iOs is missing a downward triangle/chevron icon, making the use of the dropdown confusing.

### PR DESCRIPTION
**Pull request by abbiecoghlan**
Add icon to dropdown on iOs 

closes #131 
Screenshot 2023-03-01 at 2 51 28 PM

### CODE REVIEW

**Comment by joshgachnang** [comment]
Can you add a screenshot?

**Comment by abbiecoghlan** [comment]
> Can you add a screenshot?

@joshgachnang Sorry about that! Added :)

File: packages/ui/src/SelectList.tsx [hunk 1]
```diff
@@ -34,6 +36,12 @@ export function SelectList({
   return (
     <WithLabel {...withLabelProps}>
       <RNPickerSelect
+        // Icon only needed for iOs, web and android use default icons
```
Comment (joshgachnang) [inline packages/ui/src/SelectList.tsx:39]: So weird! I wonder if it's just a silly platform decision by apple.

**Review by joshgachnang — APPROVED** [review]
Looks great!

**Pull request merged**